### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by LOGOLEX** [pail23/stiebel_eltron_isg_component#230] [Q&A]
Hello,

can someone tellme how to change Language to german?
I've already foud the "translations/de.json".

**Comment by xylone** [comment]
I think this is based on language in HA.

**Comment by Optic00** [comment]
The integration ships translations, and Home Assistant selects the language from your HA user profile rather than from an integration option. Set your profile language to German and the entity names follow. Closing as answered. Please open a new discussion if a specific string is missing or wrong.
